task_sdk/src/airflow/sdk/log.py

```diff
@@ -324,19 +324,15 @@ def configure_logging(
                 },
             },
             "loggers": {
+                # Set Airflow logging to the level requested, but most everything else at "INFO"
```

Comment:
We should cover this change in the newsfragment too, but I love it.

Reply:
I'll do this in [#45072](https://github.com/apache/airflow/issues/45072)